docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -143,6 +143,63 @@ Elasticsearch Trino
 (all others)  (unsupported)
 ============= =============
 
+.. _elasticesearch-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and Elasticsearch each support types that the other does not, this
+connector :ref:`modifies some types <type-mapping-overview>` when reading data.
```

Comment:
What does "modifies some types" mean in this context?

Reply:
In the context of a read-only connector like this, just that the Elasticsearch data types are not necessarily the types used in Trino when reading, so some modification occurs in the engine/connector.

That link goes here, to this general type mapping explanation that just got merged earlier today - https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/language/types.rst#trino-type-support-and-mapping

Reply:
I think it's misleading. The types are not "modified", in the engine or elsewhere. The types from ES get mapped into Trino types, that's all.

Reply:
@martint and @jhlodin can we change this to "maps some types"?

Reply:
I think that works for me. Just make sure whatever wording gets merged here then gets applies to all other connectors.